pkg/proto/model_service.proto

```diff
@@ -68,3 +72,20 @@ message TextRerankResponse {
     map<string, string> extra_info = 2;
     repeated float scores = 3;
 }
+
+message HighlightRequest {
+    string model = 1;
+    string query = 2;
+    repeated string documents = 3;
+    map<string, string> params = 4;
+}
+
+message HighlightResult {
+    repeated string sentences = 1;
```

Comment:
I remember we provided offset for lexical highlight right? Should match the pattern? Or how we form the E2E output